src/engine/BMGame.php

```diff
@@ -4084,7 +3851,7 @@ protected function setBMPlayerProps($property, array $value, $subclass = '') {
     /**
      * Set all players' waitingOnAction statuses to FALSE
      */
-    protected function setAllToNotWaiting() {
+    public function setAllToNotWaiting() {
```

Comment:
Yeah, but this isn't about PHP's choices, it's about our choices --- my argument is that logically it should be BMGame, not BMInterface*, which makes the decision to set a player's state to waiting or not-waiting.  I mean, again, that's not necessarily part of this refactor, but that's why it seems like a regression to me to make the function public.